scania-digital-design-system/sdds
# fix(dropdown): fix default select

**Describe pull-request**  
The dropdown option default select picks the wrong default option if one of the values is included in another.
Example:
```
export default function App() {
  const data = `[
    {"label": "Denmark","value":"1384"},
    {"label": "Sweden","value":"4"}]`;
  return (
    <div className="App">
      {/* 1384 is Denmark, but Sweden is selected in the dropdown*/}
      <sdds-dropdown-filter data={data} default-option="1384" />
    </div>
  );
}
```
See codebox for example: https://codesandbox.io/s/tegel-dropdown-test-wqzyqo?file=/src/App.tsx

**Solving issue**  
Solving github issue #493 and Azure task [AB#2427](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2427)

**How to test**  
I tested it by including the provided code example in storybook and made sure it worked, but didn't push that. 
Start with looking at the code and see if it's a potentially good solution.

**Screenshots**  
![image](https://user-images.githubusercontent.com/22123579/202654425-dbb71b52-946c-41b0-a235-e7dcadf014c2.png)
